src/erlcloud_s3.erl

```diff
@@ -467,9 +468,17 @@ extract_acl(ACL) ->
     [extract_grant(Item) || Item <- ACL].
 
 extract_grant(Node) ->
-    [{grantee, extract_user(xmerl_xpath:string("Grantee", Node))},
+    [{grantee, extract_grantee(hd(xmerl_xpath:string("Grantee", Node)))},
      {permission, decode_permission(erlcloud_xml:get_text("Permission", Node))}].
 
+extract_grantee(Node) ->
```

Comment:
are we trying to validate aws responses ourselves when parsing them (i.e. exactly one node here)? Firstly aws provides xml schema for that. Secondly, if we really need to validate xml documents on erlang side we should take more thorough and complex approach regarding that but I doubt we need it at all. 

"and the reason of use of Lists is xmerl_xpath" - I'm not sure, there are other erlcloud modules which extract data from xml node record, not from a single-element list, see erlcloud_rds